pretix_eth/management/commands/confirm_refunds.py

```diff
@@ -13,9 +13,6 @@
     OrderRefund,
 )
 
-from pretix_eth.models import (
-    WalletAddress,
-)
```

Comment:
how is confirm refunds working if we have removed this import even though the file still uses WalletAddress?

Reply:
It wasn't finished yet. There are no refunds now due to private key concerns, so refunds are handled manually